content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md

```diff
@@ -11,104 +11,241 @@ keywords:
+  * **COPY** - Select to duplicate each source file into the destination.
+    If files with the same names are present on the destination, they are overwritten.
+  * **MOVE** - Select to transfer files from the source to the destination and delete source files.
+    Copies files from the source to the destination and then deletes them from the source. Files with the same names on the destination are overwritten.
+
+### Choosing a Target Folder
+
+After choosing the direction and mode for your cloud sync task, choose the remote Google Photos folder that rclone targets to sync data.
+Because of the way rclone interacts with the Google Photos API, each target folder option has specific file management and structure requirements.
+This is due to the way rclone interacts with the Google Photos API.
+A cloud sync task cannot target the root level folder (<file>/</file>).
+
+{{< truetable >}}
+| Folder | Recommended | Direction | Description |
+|-----------|-------------|-------------|-------------|
+| <file>/album</file> | Yes | Push or Pull | Use this option for push tasks or if you prefer to organize the Google Photos library by sorting media files into one or more discrete albums. All files must be in distinct albums or child directories of the local dataset. Media files uploaded to Google Photos but not assigned to an album are not pulled to a local dataset mirroring <file>/album</file>. Files uploaded to the base level of the local dataset instead of a child directory (an album) are not pushed to <file>/album</file>. |
+|  <file>/media/all</file> | Yes | Pull | Use this option if you prefer to use the Google Photos library as single directory, without organizing media files into discrete albums. The local dataset of a <file>/media/all</file> sync task contains all media files stored on the Google Photos account at the same level, with no further organization into subdirectories. Using <file>/media/all</file> allows you to upload new files to Google Photos, without needing to organize them into albums, and then pull them to TrueNAS. |
+| <file>/upload</file> | **No** | Push | Media files pushed from the local dataset to <file>/upload</file> are then uploaded to Google Photos and not sorted into an album. Because <file>/upload</file> is a temporary storage location, it can not accurately synchronize from one task to the next. Pushing to this folder does not preserve metadata and can result in duplicated files, poor performance, file name instability. |
+{{< /truetable >}}
+
+### Selecting the Dataset and Organizing Files
+
+Next, select TrueNAS SCALE local dataset or create a new one to use as the source or destination.
+
+{{< expand "Creating a Dataset" "v" >}}
+{{< include file="/static/includes/CreateDatasetSCALE.md" >}}
+{{< /expand >}}
+
+Configure the local dataset file structure (for push tasks) or Google Photos library albums (for pull tasks) as required by your direction, mode, and target selections (see above).
```

Comment:
I think it would be better to put line 74 as the first line in this section as it is the important part about creating or selecting the dataset. What do you think about this?
```suggestion
Configure the local dataset file structure (for push tasks) or Google Photos library albums (for pull tasks) as required by your direction, mode, and target selections (see above).
Either select the TrueNAS SCALE local dataset or create a new one to use as the source or destination.

{{< expand "Creating a Dataset" "v" >}}
{{< include file="/static/includes/CreateDatasetSCALE.md" >}}
{{< /expand >}}
```

Reply:
I was thinking more in terms of how you manage files within the dataset after selecting or creating it. What do you think about this instead:

```suggestion
Next, select TrueNAS SCALE local dataset or create a new one to use as the source or destination.

{{< expand "Creating a Dataset" "v" >}}
{{< include file="/static/includes/CreateDatasetSCALE.md" >}}
{{< /expand >}}

Configure file management structure inside the local dataset (for push tasks) or albums in the Google Photos (for pull tasks) as required by your direction, mode, and target selections (see above).
```

Reply:
Good idea. With that change I'm okay with it being here under the dataset expand, but still remove Next from line 68